src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -862,6 +907,22 @@
     justify-content: center;
     align-items: stretch;
 
+
+
+    input[type='text'] {
+      background: none !important;
+      background-color: #ffffff;
```

Comment:
how does this translate in the dark theme?

Reply:
Ah, good point! I will ask @lil-kim for dark theme colors and update.